.github/workflows/test-EMULATOR.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       uses: nschloe/action-cached-lfs-checkout@v1.2.1
       with:
         submodules: recursive
-        lfs: 'true'
+        # lfs: 'true'
```

Comment:
this is dead code? so please remove.

Reply:
I would like to let it stay, as the line above (`# uses: actions/checkout@v3 # incompat`) is also there for documenting that we are now using another action. The `lfs` is strictly required for the commented variant. Do you think we should still remove it?

Reply:
@simonmicro OOOOH It's not a dead code!!!!, I want to leave `# blah blah`comments for documentation in a good format.. in a better format so that it doesn't look like a dead code...